src/Symfony/Bridge/Doctrine/Tests/DataFixtures/ContainerAwareLoaderTest.php

```diff
@@ -16,6 +16,11 @@
 use Symfony\Bridge\Doctrine\Tests\Fixtures\ContainerAwareFixture;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
+/**
+ * @group legacy
+ *
+ * @deprecated since Symfony 6.4, to be removed in 7.0
```

Comment:
can be removed as it is easy to spot once the deprecated thing it covers gets removed. Same for all fixtures/tests

Reply:
Alright, remove! 🙂